src/control/server/ctl_storage_rpc.go

```diff
@@ -6,22 +6,40 @@
 
 package server
 
+/*
+#include "daos_srv/control.h"
+*/
+import "C"
```

Comment:
We have generally been keeping cgo in specific packages rather than sprinkling it around the codebase. This makes it easier to understand package boundaries and debug build problems. Please define Go consts based on the C consts in the `drpc` package and use those.

NB: I have a very large and disruptive PR that creates a new `lib/daos` package for these non-dRPC consts: https://github.com/daos-stack/daos/pull/8359 ... I plan to land this after 2.2 has been branched from master, and I'll pull these new consts into that package then as well.

Reply:
- [ ] Move C const declaration of `DAOS_MD_CAP_ENV`, `DEFAULT_DAOS_MD_CAP_SIZE`, etc. to `drpc`